rougelike_game/Game_working.py

```diff
@@ -1,76 +1,157 @@
-import subprocess as sp
+from Map_handler import *
 from Player import *
-
+import random
+import webbrowser
+import time
+import keyboard
+import subprocess as sp
 EMPTY_FIELD = '.'
 level = []
+e = 0
+again = ''
 
+b1 = 1
+b2 = 1
+level = handle_level(level)
+looking = '@'
```

Comment:
why do you need `looking` variable? you can invoke the function like:
`player1 = find_player(level, Player, '@')`